npm/pkg/dataplane/ipsets/ipsetmanager.go

```diff
@@ -463,12 +469,20 @@ func (iMgr *IPSetManager) ApplyIPSets() error {
 	defer metrics.RecordIPSetExecTime(prometheusTimer) // record execution time regardless of failure
 	err := iMgr.applyIPSets()
 	if err != nil {
-		metrics.SendErrorLogAndMetric(util.IpsmID, "error: failed to apply ipsets: %s", err.Error())
+		// exponentially increase maxRestoreTryCount
+		iMgr.maxRestoreTryCount *= 2
```

Comment:
Why do we want to exponentially increase here?

Reply:
this just results in 10 tries, we can just remove this and change the largestretry to 10

Reply:
discussed offline. setting a constant limit of 5